yarn.lock

```diff
@@ -17824,7 +17824,7 @@ typedoc@^0.15.6:
     typedoc-default-themes "^0.6.3"
     typescript "3.7.x"
 
-typescript@3.7.x, typescript@^3.0.3, typescript@^3.6.4, typescript@^3.7.2, typescript@^3.7.4, typescript@^3.7.5:
+typescript@3.7.x, typescript@^3.0.3, typescript@^3.6.4, typescript@^3.7.4, typescript@^3.7.5:
```

Comment:
seems we have a lot of versions of `typescript` but it's been around for a while and never caused any problems so far

Reply:
No, they all resolve to 3.7.5 (see the following line in that file)

Reply:
ah, nvm, need more maté